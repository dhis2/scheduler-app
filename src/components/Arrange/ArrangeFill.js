import React from 'react'
import { node } from 'prop-types'
import styles from './ArrangeFill.module.css'

/**
 * Fills all available space
 */

const ArrangeFill = ({ children }) => {
    return <div className={styles.fill}>{children}</div>
}

ArrangeFill.propTypes = {
    children: node.isRequired,
}

export default ArrangeFill
