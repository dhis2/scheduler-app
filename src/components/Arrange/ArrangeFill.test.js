import React from 'react'
import { shallow } from 'enzyme'
import ArrangeFill from './ArrangeFill'

describe('<ArrangeFill>', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<ArrangeFill>Text</ArrangeFill>)

        expect(wrapper).toMatchSnapshot()
    })
})
