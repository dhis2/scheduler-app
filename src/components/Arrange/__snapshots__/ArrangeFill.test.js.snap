// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrangeFill> renders correctly 1`] = `
<div
  className="fill"
>
  Text
</div>
`;
